src/ideabox/policy/viewlets/footer.pt

```diff
@@ -1,6 +1,6 @@
 <script type="text/javascript">
 // Sorry for this but I am waiting for this: https://github.com/plone/Products.CMFPlone/issues/1674
-if (ploneFormTabbing) {
+if (typeof ploneFormTabbing !== 'undefined' && ploneFormTabbing) {
```

Comment:
I would add a subscriber to document ready to ensure that this code will run if ploneFromTabbing is available.

Reply:
https://github.com/IMIO/ideabox.policy/commit/5c3b6fcaafbb218f7aca3b09ad5521af87c2c94a

Reply:
Fixed ! : 
https://github.com/IMIO/ideabox.policy/commit/ca1ebfd4734c98554cddb0d9ad62f3aae31b6c8e